Hotfix/redirect (#65)
WeirdAlex03/Project-EverGreen: README.md

# Redirects

> Please note that requests to port 8080 are redirected to port 80 or 443 as static files are hosted through nginx. Port 443 is used in instances where certs are present. Port 80 is used in instances where they are not present, in which case they are then upgraded to 443 with certs.

Reviewer: Last part is a little clunky to me
```suggestion
Please note that requests to port 8080 are redirected to port 80 or 443 as static files are hosted through nginx. Port 443 is used in instances where certs are present. Port 80 is used in instances where they are not present. Requests to port 80 will be upgraded to HTTPS (443) if that is in use.
```
